testing/bsg_dataflow/bsg_fifo_1r1w_large/Makefile

```diff
@@ -20,14 +20,9 @@ IO_MASTER_1_PERIOD =   2
 CORE_1_PERIOD      =   1
 endif
 
+export BSG_CADENV_DIR = $(abspath .)/../../../../bsg_cadenv
```

Comment:
A little more flexible:

TOP ?= $(shell git rev-parse --show-toplevel)
BSG_CADENV_DIR ?= $(TOP)/../bsg_cadenv


No need for export